src/uthash.h

```diff
@@ -177,24 +193,45 @@ do {
 
 #define HASH_MAKE_TABLE(hh,head)                                                 \
 do {                                                                             \
+  int _mem_ok = 1;                                                               \
   (head)->hh.tbl = (UT_hash_table*)uthash_malloc(sizeof(UT_hash_table));         \
-  if (!(head)->hh.tbl) {                                                         \
+  if (!((head)->hh.tbl)) {                                                       \
```

Comment:
Do you need these extra parentheses to suppress a compiler warning or something, or is this just a merge conflict?